portfolio/src/main/webapp/script.js

```diff
@@ -131,6 +150,21 @@ async function deleteComments() {
   }
 }
 
+/**
+ * Returns the toxicity of the text input.
+ */
+async function getToxicity(text) {
+  const response = await fetch(
+  'https://commentanalyzer.googleapis.com/v1alpha1/comments:analyze?key=AIzaSyC9mw1TrAmKo3FHkkEP0gkoDLQ-_lIFpLI',
+  {
+    method: 'POST',
+    headers: {'Content-Type': 'application/json'},
+    body: JSON.stringify({comment: {text: text}, languages: [], requestedAttributes: { TOXICITY: {} }})
+  });
```

Comment:
Just FYI for the future, you might want to handle cases where the Perspective API fails (maybe because the server is down, or the user submits a comment in an unsupported language).  You don't have to worry about this for this PR, but in later projects we'll probably want to handle these errors

Reply:
Noted, I'll keep this in mind!